src/graph_gui.cpp

```diff
@@ -339,8 +340,13 @@ struct BaseGraphWindow : Window {
 		/* Don't draw the first line, as that's where the axis will be. */
 		x = r.left + x_sep;
 
-		for (int i = 0; i < this->num_vert_lines; i++) {
-			GfxFillRect(x, r.top, x, r.bottom, GRAPH_GRID_COLOUR);
+		int grid_colour = GRAPH_GRID_COLOUR;
+		for (int i = 1; i < this->num_vert_lines + 1; i++) {
+			/* If using real-time, we separate periods with a lighter line. */
```

Comment:
Nitpick: you name "real-time" many things throughout this PR. "real-time mode", "real-world minutes", and now "real-time". Maybe a good idea to go over it once and sync up all the wording?